src/Runner.Worker/ExecutionContext.cs

```diff
@@ -1355,6 +1355,10 @@ private static void ResolvePathsInExpressionValuesDictionary(this IExecutionCont
         {
             foreach (var key in dict.Keys.ToList())
             {
+                if (key == PipelineTemplateConstants.HostWorkspace)
+                {
+                    continue;
```

Comment:
```suggestion
                if (key == PipelineTemplateConstants.HostWorkspace)
                {
                    // The HostWorkspace context var is excluded so that there is a var that always points to the host path. 
                    // This var can be used to translate back from container paths, e.g. in HashFilesFunction, which always runs on the host machine
                    continue;
```